docs/source/tutorials/fashion_mnist.ipynb

```diff
@@ -0,0 +1,1257 @@
+    "    train_id_list.append(train_ids)\n",
+    "    test_id_list.append(test_ids)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Compute out-of-sample predicted probabilities and feature embeddings"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pred_probs_list, embeddings_list = [], []\n",
+    "\n",
+    "for i in range(K):\n",
+    "    print(f\"Training on fold: {i+1}\")\n",
+    "\n",
+    "    # Create train and test sets and corresponding dataloaders\n",
+    "    trainset = transformed_dataset.select(train_id_list[i])\n",
+    "    testset = transformed_dataset.select(test_id_list[i])\n",
+    "\n",
+    "    trainloader = DataLoader(trainset, batch_size=batch_size, shuffle=True, num_workers=2)\n",
+    "    testloader = DataLoader(testset, batch_size=batch_size, shuffle=False, num_workers=2)\n",
+    "\n",
+    "    # Train model\n",
+    "    model = train(trainloader, testloader, n_epochs, patience)\n",
+    "\n",
+    "    # Compute out-of-sample embeddings\n",
+    "    print(\"\\nComputing feature embeddings...\")\n",
+    "    fold_embeddings = compute_embeddings(model, testloader)\n",
+    "    embeddings_list.append(fold_embeddings)\n",
+    "\n",
+    "    print(\"\\nComputing predicted probabilities...\")\n",
+    "    # Compute out-of-sample predicted probabilities\n",
+    "    fold_pred_probs = compute_pred_probs(model, testloader)\n",
+    "    pred_probs_list.append(fold_pred_probs)\n",
+    "\n",
+    "print(\"Finished Training\")\n",
+    "\n",
+    "\n",
+    "# Combine embeddings and predicted probabilities from each fold\n",
+    "features = torch.vstack(embeddings_list).numpy()\n",
+    "\n",
+    "logits = torch.vstack(pred_probs_list)\n",
+    "pred_probs = nn.Softmax(dim=1)(logits).numpy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Reorder rows of the dataset based on row-order in `features` and `pred_probs`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "indices = np.hstack(test_id_list)\n",
+    "dataset = dataset.select(indices)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 7. Use cleanlab to find issues\n",
+    "\n",
+    "Based on the given labels, out-of-sample predicted probabilities and feature embeddings, cleanlab can quickly help us detect potential issues in our dataset. \n",
+    "\n",
+    "Here, we use cleanlab's `Datalab` class to find potential issues in our data. `Datalab` has several ways of loading the data. For our current scenario, we are employing the huggingface dataset. `Datalab` takes in two optional arguments, `label_name`, which corresponds to the label column's name, and `image_key`, corresponding to the image column's name."
```

Comment:
```suggestion
    "Here, we use cleanlab's `Datalab` class to find potential issues in our data. `Datalab` has several ways of loading the data. For our current scenario, we are employing the huggingface dataset. `Datalab` takes in two optional arguments, `label_name`, which corresponds to the label column's name, and `image_key`, corresponding to the image column's name. When you provide these optional arguments, `Datalab` will audit your dataset for more types of issues than it would by default."
```